runtime/runtime-params-estimator/src/stats.rs

```diff
@@ -26,8 +26,11 @@ impl Measurements {
         block_size: usize,
         block_cost: ExecutionCost,
     ) {
+        #[cfg(feature = "costs_counting")]
         let ext_costs = node_runtime::EXT_COSTS_COUNTER
             .with(|f| f.borrow_mut().drain().collect::<HashMap<_, _>>());
+        #[cfg(not(feature = "costs_counting"))]
+        let ext_costs = HashMap::new();
```

Comment:
What is expected semantics here? Guess without costs counting this code shalln't make much sense.

Reply:
I don't know, this is just the simplest patch that makes this compilable, without introducing unused/unreachable warnings )

I think the right high-level semantics would be to print "cost counting is disabled" when trying to *get* the metrics, but I want to tackle that together with removing the cargo feature (I am prepping a little [likbez](https://en.wikipedia.org/wiki/Likbez) for that...)

Reply:
Simply exit when cost_counting is not enabled?

Reply:
or exit much earlier in param estimator when compile without cost counting, and unreachable here. what exactly we want is to enforce "it must compile with cost counting"

Reply:
> "it must compile with cost counting"

This should do it: aa21023cae90142226ae618f2e5dbef96885fef6